pages/docs/tutorials/configure-asyncapi-kafka.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Configuring Schemas for Kafka messages using AsyncAPI
+description: In this tutorial, you'll learn how to configure an AsyncAPI document for Kafka messages.
+weight: 80
+---
+
+## Introduction
+
+In this tutorial, you will learn how to create a schema document for Kafka Messages using AsyncAPI. Additionally, you will also learn about event-driven architecture, pub/sub model, and brokers in relation to Kafka. 
+  
+Let’s assume, you have a service that publishes data to a Kafka topic whenever a new user signs up into the application.
+
+Basically, you want to build a system that can track the users who have signed up for your application, the entire flow of the application would be:
+
+```mermaid
+flowchart TD
+    Service[Service] -->|Publishes Event| Broker[Message Broker]
+    App[Your Application] -->|Connects to a topic| Broker
+    Broker -->|Sends Event Stream| App
+```    
+
+
+## Background context
+
+[Event-driven architecture (EDA)](/docs/tutorials/getting-started/event-driven-architectures) is a design pattern that revolves around the production, detection, and reaction to events over time. It consists of three main components: a message broker, event publishers, and subscribers, which together serve as the backbone for event exchange within different services. 
+
+[Message brokers](/docs/tutorials/getting-started/event-driven-architectures#message-broker) facilitate asynchronous communications between services, meaning that the sending service doesn't have to wait for the receiving service's response. This allows multiple services to talk to each other directly, even if they are written in different languages. A great example of such a message broker is Apache Kafka. [Apache Kafka](https://kafka.apache.org/) is a distributed system that is used by thousands of companies for their event-driven system.
+
+[Publish/Subscribe (Pub/Sub)](/docs/tutorials/getting-started/event-driven-architectures#publishersubscriber) is a typical model in EDAs, providing flexible coupling between publishers and subscribers. In this model, the publishers of event notifications act as publishers and the consumers act as corresponding subscribers.
+
+## Creating AsyncAPI document for Kafka
+
+In this section, you’ll create an AsyncAPI document to describe the `UserSignUp` API. The same document can be later used to generate code and documentation as per requirement. 
+
+### Define AsyncAPI Version, API Information, and Server
+
+Initially, you need to describe your application, including the AsyncAPI version, the info about the document, and the server your application is based upon.
+
+```
+asyncapi: "3.0.0"
+info:
+  title: "User Signup API"
+  version: "1.0.0"
+  description: |
+    The API notifies you whenever a new user signups into the application
+servers:
+  kafkaServer:
+    host: test.mykafkacluster.org:8092
+    description: "Kafka Server"
+    protocol: kafka
```

Comment:
Double quotes are not needed when the value is a string. You're using them sometimes and sometimes not so I'd rather avoid using them to improve the reading experience.

```suggestion
asyncapi: 3.0.0
info:
  title: User Signup API
  version: 1.0.0
  description: The API notifies you whenever a new user signs up in the application.
servers:
  kafkaServer:
    host: test.mykafkacluster.org:8092
    description: Kafka Server
    protocol: kafka
```